x/profiles/keeper/alias_functions.go

```diff
@@ -75,3 +75,49 @@ func (k Keeper) IterateUserRelationships(ctx sdk.Context, user string, fn func(i
 		i++
 	}
 }
+
+// --------------------------------------------------------------------------------------------------------------------
+
+// IterateChainLinks iterates through the chain links and perform the provided function
+func (k Keeper) IterateChainLinks(ctx sdk.Context, fn func(index int64, link types.ChainLink) (stop bool)) {
+	store := ctx.KVStore(k.storeKey)
+
+	iterator := sdk.KVStorePrefixIterator(store, types.ChainLinksPrefix)
+	defer iterator.Close()
+
+	i := int64(0)
+
+	for ; iterator.Valid(); iterator.Next() {
+		link := types.MustUnmarshalChainLink(k.cdc, iterator.Value())
+
+		stop := fn(i, link)
+
+		if stop {
+			break
+		}
+
+		i++
+	}
```

Comment:
Just some small changes to remove empty lines
```suggestion
	i := int64(0)
	for ; iterator.Valid(); iterator.Next() {
		link := types.MustUnmarshalChainLink(k.cdc, iterator.Value())

		stop := fn(i, link)
		if stop {
			break
		}
		i++
	}
```